src/store/reducers/login.js

```diff
@@ -22,7 +23,7 @@ export default combineReducers({
   },
 
   error: (state = null, { type, error, payload }) => {
-    if (error && (type === LOGIN || type === SIGNUP || type === VERIFY_EMAIL || type === SEND_EMAIL_VERIFICATION)) return payload.message
```

Comment:
Might be nice to do an `.includes()` on this line as well while you're in there. That said also all for letting sleeping dogs lie as I know changes require some leave of sanity check testing...